pkg/cmd/cli/cmd/volume.go

```diff
@@ -0,0 +1,588 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	kapierrors "github.com/GoogleCloudPlatform/kubernetes/pkg/api/errors"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/meta"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	kerrors "github.com/GoogleCloudPlatform/kubernetes/pkg/util/errors"
+
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const (
+	volumeLong = `Update volumes on a pod
+
+This command can add, update, list or remove volumes from containers in pods or
+any object that has a pod template (replication controllers or deployment configurations).
+You can specify a single object or multiple, and alter volumes on all containers or
+just those that match a wildcard.`
+
+	volumeExample = `  // Add new volume of type 'emptyDir' for deployment 'registry' and mount under /opt inside the containers
+  $ %[1]s volume dc/registry --add --mount-path=/opt
+  Here name of the volume is auto generated and default type 'emptyDir' will be used.
```

Comment:
Move this up above the command, prefix with `//`: `// The volume name is auto generated`. I don't think you need to mention `emptyDir` again as it's in the 1st comment line